index.js

```diff
@@ -260,7 +260,7 @@ module.exports = {
     if (this._emberVersionRequiresModulesAPIPolyfill()) {
       const ModulesAPIPolyfill = require('babel-plugin-ember-modules-api-polyfill');
 
-      return [[ModulesAPIPolyfill]];
+      return [[ModulesAPIPolyfill, { blacklist: ['@ember/debug'] }]];
```

Comment:
are https://github.com/ember-cli/ember-rfc176-data/blob/master/globals.json#L74-L80 all supported in the other Babel plugin?